### PR TITLE
Copy improvements for readme and Quickstart

### PR DESCRIPTION
**Pull request by ow**
Hot on the heels of #854, I've done a more in-depth pass over both `readme` and `quickstart` to help provide clarity/assumed context for newer developers using this guide for the first time (particularly those that may not have the required pre-requisites on their machine or may not have a Heroku account). 

CCing the same Shopify folks from #854 since they have context already. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ After running the `install` generator, you can start your app with `bundle exec
 $ rails generate shopify_app:home_controller
 ```
 
-This generator creates an example home controller and view which fetches and displays products using the ShopifyAPI
+This generator creates an example home controller and view which fetches and displays products using the Shopify API
```
Comment (gfscott) [inline README.md]: Would be great to be specific that this is the **Shopify Admin API**

File: README.md [hunk 2]
```diff
@@ -408,9 +408,11 @@ strategy.options[:old_client_secret] = ShopifyApp.configuration.old_secret
 App Tunneling
 -------------
 
-Your local app needs to be accessible from the public Internet in order to install it on a shop, use the [App Proxy Controller](#app-proxy-controller-generator) or receive Webhooks. Use a tunneling service like [ngrok](https://ngrok.com/), [Forward](https://forwardhq.com/), [Beeceptor](https://beeceptor.com/), [Mockbin](http://mockbin.org/), [Hookbin](https://hookbin.com/), etc.
+Your local app needs to be accessible from the public Internet in order to install it on a Shopify store, to use the [App Proxy Controller](#app-proxy-controller-generator) or receive Webhooks. 
+
+Use a tunneling service like [ngrok](https://ngrok.com/), [Forward](https://forwardhq.com/), [Beeceptor](https://beeceptor.com/), [Mockbin](http://mockbin.org/), [Hookbin](https://hookbin.com/) to make your development environment accessible to the internet.
```
Comment (gfscott) [inline README.md]: needs an "or" before the last comma-separated item.

File: docs/Quickstart.md [hunk 3]
```diff
@@ -17,71 +19,85 @@ $ git commit -m 'new rails app'
...
+# Use the keys from your app you created in the partners area
 $ rails generate shopify_app --api_key <shopify_api_key> --secret <shopify_api_secret>
 $ git add .
 $ git commit -m 'generated shopify app'
 ```
 
 If you forget to set your keys or redirect uri above, you will find them in the shopify_app initializer at: `/config/initializers/shopify_app.rb`.
 
-We recommend adding a gem or utilizing ENV variables to handle your keys before releasing your app.
+We recommend adding a gem or utilizing environment variables (ENV) to handle your keys before releasing your app. [Learn more about using environment variables.](https://www.honeybadger.io/blog/ruby-guide-environment-variables/)
```
Comment (gfscott) [inline docs/Quickstart.md]: "(ENV)" is a little ambiguous now, and we referenced the lowercase name of the file above. Maybe instead:

> ...utilizing an environment variables file (`.env`) to handle your keys...

File: docs/Quickstart.md [hunk 4]
```diff
@@ -17,71 +19,85 @@ $ git commit -m 'new rails app'
...
+
+Ensure you have created a [development store](https://help.shopify.com/en/api/getting-started/making-your-first-request#create-a-development-store) using the Shopify Partner Dashboard. If you don't already have one, [create one by following these instructions](https://help.shopify.com/en/api/getting-started/making-your-first-request#create-a-development-store).
 
 ##### Note: The following step will cause your store to become `transfer-disabled.` Read more about store transfer and why it's important [here](https://help.shopify.com/en/api/guides/store-transfers#transfer-disabled-stores). This is an irreversible change, so be sure you don't plan to transfer this store to a merchant.
 
-Install the app using the Partner Dashboard in the Apps area.
+Install the app onto your new development store using the Partner Dashboard by heading to the apps page, clicking your app, and looking for the `Test your app` instructions where you can select a store to install your app on.
```
Comment (gfscott) [inline docs/Quickstart.md]: "heading to" is an English idiom, we could just say "opening the apps page"

(if only we could link directly to their apps index!)

File: README.md (hunk 2 above)
Comment (ow) [reply README.md]: Nice, thanks

**Pull request merged**